charts/templates/statefulset.yaml

```diff
@@ -168,10 +170,12 @@ spec:
                   - |
                     log () {
                       local TIMESTAMP=$(date +"%Y-%m-%d %T.%3N")
-                      echo "${TIMESTAMP} $@" > /proc/$pid/fd/1
+                      if [ -n "$pid" ]; then
```

Comment:
Ideally we would not be cutting and pasting this log code around. We can keep it for now but we should look at options to have some shared functions for our scripts. Perhaps use a config map to mount them.

Reply:
Agree we should have one common function and use it as reference. Will create ticket to address it post 1.0.0 release